server/domain/devices/user.go

```diff
@@ -0,0 +1,24 @@
+package devices
+
+import "github.com/jinzhu/gorm"
+
+type User struct {
+	gorm.Model
+	Email    string `json:"email"`
+	Password string `json:"password"`
```

Comment:
Are passwords stored as plain text?

Reply:
yes, for the sake of simplicity, i hope nobody wants to hack our local database) or do you think i need to add encryption?

Reply:
Yes. I think storing some password hashes instead of passwords would be enough